### DISCUSSION TITLE
Benchmarking general ideas

### DISCUSSION THREAD

**Discussion by Daniel-Ibarrola** [uibcdf/OpenPharmacophore-Tools#2] [Ideas]
Let us use this discussion to see which could be a possible structure to test OpenPharmacophore and see if we can reproduce the results.

**Comment by Daniel-Ibarrola** [comment]
In [Wolber2005LigandScout_ce](https://doi.org/10.1021/ci049885e) they built a pharmacophore for Human Rhinovirus Serotype 16 Inhibitors. They used the following PDB entries. This are for the protein-ligand complex are:

•	http://www.rcsb.org/structure/1NCR
•	http://www.rcsb.org/structure/1ND3
•	http://www.rcsb.org/structure/1C8M

PDB for the ligand:
•	http://www.rcsb.org/ligand/W11

They also generated a pharmacophore model for Tyrosine Kinase Inhibitors. They used this PDB entries:
•	http://www.rcsb.org/structure/1fpu
•	http://www.rcsb.org/structure/1fpu
•	http://www.rcsb.org/structure/1OPJ

Ligand Scout was used to build the models.

**Comment by Daniel-Ibarrola** [comment]
In Thesis:Sydow2015 a pharmacophore model was created for sulfotransferase, ([PDB](http://www.rcsb.org/structure/1HY3)). They used the ligands kaempferol or 4-OH-HCB. Moreover, they used MD simulations to generate the models.

In Thesis:Michelarakis2019 pharmacophores derived from MD simulations were generated for  sets of different proteins. The PDB entries can be found in their work. 

Both use MD simulations, so I believe they could be useful for testing when the project is more advanced.

**Comment by Daniel-Ibarrola** [comment]
In [Dror2009Novel_cd](https://doi.org/10.1021/ci900263d)  pharmacophore models were created for alpha 1A, a G-Protein Coupled Receptor. 

They used the free software PharmaGist. Perhaps it can be used for comparisons.

**Comment by dprada** [comment]
I will add another one. As far as I know, one of the firsts tries, or the first, to build a pharmacophore model from molecular dynamics, and it is structure-based. 

[Carlson, Heather A, Kevin M Masukawa, Kathleen Rubins, Fredric D Bushman, William L Jorgensen, Roberto D Lins, James M Briggs, and J Andrew McCammon. “Developing a Dynamic Pharmacophore Model for HIV-1 Integrase.” Journal of Medicinal Chemistry 43, no. 11 (2000): 2100–2114.](https://doi.org/10.1021/jm990322h).

**Comment by dprada** [comment]
> Let us use this discussion to see which could be a possible structure to test OpenPharmacophore and see if we can reproduce the results.

I have changed the name of the discussion to extend the topic to what could be the set of molecular systems, structures, ligands... to:
- test OpenPharmacophore
- see if we can reproduce published results
- make comparable benchmarks to see if we are doing good or bad.
